pkg/monitoring/system/collector.go

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package system
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/tools/cache"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/log"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/guestfs"
+)
+
+var virtualizationEnabled = prometheus.NewDesc(
+	"kubevirt_node_virtualization_status",
```

Comment:
@machadovilaca Can we try to add this as a label to the node info metric?

Reply:
I don't think they will accept it, the label that I use to verify it has the extensions is from "kubevirt.io/kubevirt/pkg/virtctl/guestfs"